testng-webdriver/README.md

```diff
@@ -1,3 +1,5 @@
 # TestNG Webdriver
 
-[![Maven central](https://maven-badges.herokuapp.com/maven-central/com.dev9/testng-webdriver/badge.svg)](http://mvnrepository.com/artifact/com.dev9/testng-webdriver)
\ No newline at end of file
+[![Maven central](https://maven-badges.herokuapp.com/maven-central/com.dev9/testng-webdriver/badge.svg)](http://mvnrepository.com/artifact/com.dev9/testng-webdriver)
+
+You can use the TestNg Suite XML file to specify which browser you would like by setting the following field 'webtest.profile'.
```

Comment:
Please specify all supported browsers and the expected value for webtest.profile